### ISSUE TITLE
ResponseValidationError

### ISSUE THREAD

**Issue by domenkozar** [flakestry/flakestry.dev#46]
ResponseValidationError: 1 validation errors:
  {'loc': ('response',), 'msg': 'none is not an allowed value', 'type': 'type_error.none.not_allowed'}

**Comment by lorenzleutgeb** [comment]
Looks like we're [hitting this](https://github.com/ngi-nix/ngipkgs/actions/runs/8930687089/job/24531280533#step:2:836)? Any fixes?

**Comment by domenkozar** [comment]
Do you have an idea what's causing this?

**Comment by lorenzleutgeb** [comment]
Nope. Got another one though, in a repo with a single commit: https://github.com/lorenzleutgeb/radicle-nix/actions/runs/9167968223/job/25205978880 (don't be confused about it being green, I set `continue-on-error` because I was expecting it...).

Both of them are attempts for a "rolling release", check `.github/workflows/flakestry.yaml`s:

 - [ngi-nix/ngipkgs](https://github.com/ngi-nix/ngipkgs/blob/24a4fe2f4a159cca88a162adc85f09ed7b3de32d/.github/workflows/flakestry.yaml)
 - [lorenzleutgeb/radicle-nix](https://github.com/lorenzleutgeb/radicle-nix/blob/d3f4f21164a3a58d77d13bd9d1fe263ae7ddc3f9/.github/workflows/flakestry.yaml)
